composer.json

```diff
@@ -1,30 +1,38 @@
 {
   "name": "spryker-eco/fact-finder-sdk",
+  "type": "library",
   "description": "FactFinderSdk module",
-  "license": "MIT",
+  "license": "proprietary",
```

Comment:
This doesnt fit the other files and licence doc block now.
Please be careful with such legal changes, as they can have serious consequences.

Reply:
That's a change from `dev:composer:update-json-files` I didn't notice.